internal/service/ipam/model_ipamsvc_host.go

```diff
@@ -33,6 +34,25 @@ type IpamsvcHostModel struct {
 	Type             types.String `tfsdk:"type"`
 }
 
+type IpamsvcHostModelWithRetryAndTimeouts struct {
```

Comment:
since we have now two different models that cannot be reused for resource and data source, can we move the respective models and schemas to be with in the resource file itself, instead of being in `model_ipamsvc_host`

Reply:
I have deleted the model file and split the Schema in to Resource and Data source files.